packages/gestalt-datepicker/src/DatePicker.js

```diff
@@ -38,9 +39,24 @@ type Props = {|
   rangeEndDate?: Date,
   rangeSelector?: 'start' | 'end',
   rangeStartDate?: Date,
+  timezone?: Timezone,
   value?: Date,
 |};
 
+export const getDisplayDate = (date: Date, timezone: Timezone): Date => {
+  if (timezone === 'local') {
+    return date;
+  }
+  return new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate());
+};
+
+export const getUpdatedDate = (date: Date, timezone: Timezone): Date => {
+  if (timezone === 'local') {
+    return date;
+  }
+  return new Date(Date.UTC(date.getFullYear(), date.getMonth(), date.getDate()));
```

Comment:
can we invert this? 

```
if (timezone === 'utc') {
  return new Date(Date.UTC(date.getFullYear(), date.getMonth(), date.getDate()))
}

return date;

```

I'm just thinking if we pass for whatever reason the incorrect prop value 'locol' rather than 'local' (which it should be caught by Flow, but in case we don't have Flow coverage!) then we would be applying 'utc' instead. So I would set the conditional logic to 'utc'